src/ui/button/button.jsx

```diff
@@ -137,16 +142,16 @@ export function AuthButton(props : ButtonProps) : ElementNode {
                 onClick={ clickHandler }
                 onKeyPress={ keypressHandler }
                 tabindex='0'>
-
-                <div class={ CLASS.BUTTON_LABEL }>
-                    <PPSymbol style={ style } />
-                    <Logo style={ style } />
-                    {
-                        fundingSource === FUNDING.CREDIT ?
-                            <PPCreditLogo style={ style } locale={ locale } /> :
-                            <Label style={ style } content={ content } />
-                    }
-                </div>
+                    
+                { customLabel &&
+                    <div class={ CLASS.BUTTON_LABEL }>
+                        <PPSymbol style={ style } />
+                        {
+                            fundingSource === FUNDING.CREDIT ?
+                                <PPCreditLogo style={ style } locale={ locale } /> :
+                                <Label style={ style } content={ customLabel !== undefined ? customLabel : 'Connect with PayPal' } />
```

Comment:
Can you help me understand what `customLabel` is -- is there a spec or doc for this?

We should also not hard-code content like `Connect with PayPal` as the locale may not be `en`.

Reply:
i have removed the connect with PayPal. Custom label is dynamically generated by customer input, with locale and inputlabel type, if user says 1. locale : FR , inputLabel : connect  --->  Connexion avec PayPal